src/components/posts/PostForm/PostForm.tsx

```diff
@@ -21,10 +22,19 @@ import { PostFormFragment } from './PostForm.generated';
 export type PostFormProps<T, R = void> = DialogFormProps<T, R> & {
   /** The pre-existing post to edit */
   post?: PostFormFragment;
+  disableMembership: boolean;
 };
 
+const shareabilityList = (disableMembership: boolean) =>
+  without(
+    PostShareabilityList,
+    'ProjectTeam',
```

Comment:
`PostShareabilityList` shouldn't include `ProjectTeam` anymore since it's been deprecated.